### PR TITLE
feat(docs): Migrate tabs docs

### PR DESCRIPTION
**Pull request by bry4n**
JIRA: https://jira.rax.io/browse/FRMW-204

* [x] Dev LGTM
* [x] Dev LGTM

### CODE REVIEW

**Comment by CITguy** [comment]
Looks like protractor 2.5.0 dropped and is causing havoc. I'll submit a PR to pin to protractor 2.4.0

**Comment by 100stacks** [comment]
@CITguy :ok_hand: :+1: I am getting the same...

**Comment by CITguy** [comment]
Restarted travis job considering that we've updated to node 0.12

**Comment by CITguy** [comment]
Nope, @bry4n. please rebase.

**Comment by bry4n** [comment]
@CITguy rebased :+1:

**Comment by 100stacks** [comment]
LGTM

File: src/tabs/tabs.js [hunk 1]
```diff
@@ -4,6 +4,21 @@
...
+ * ## Usage
+ *
+ * Usage is the exact same as demoed on the Angular-UI Bootstrap site. See
+ * [the Angular-UI Bootstrap Docs](http://angular-ui.github.io/bootstrap/) for
+ * further guidance on usage and configuration of this component.
+ *
+ * ## Disclaimer
+ *
+ * Only the default horizontal tabs are supported by these styles. `Vertical`,
+ * `Pills` and `Justified` tabs are currently unsupported. Support can be added
```
Comment (CITguy) [inline src/tabs/tabs.js]: Please remove "Support can be added ..." sentence.

Comment (bry4n) [inline src/tabs/tabs.js]: removed

**Comment by CITguy** [comment]
LGTM

**Pull request merged**